src/clj/ataru/virkailija/virkailija_routes.clj

```diff
@@ -389,6 +391,39 @@
 
     (api/context "/background-jobs" []
       :tags ["background-jobs-api"]
+
+      (api/POST "/start-kk-application-payment-maksut-poller-job" {session :session}
+        :path-params []
+        :summary "Triggers a job for updating maksut status for all open higher education application payments"
+        (if (get-in session [:identity :superuser])
+          (do (kk-application-payment-maksut-poller-job/start-kk-application-payment-maksut-poller-job
+                job-runner)
+              (response/ok {}))
+          (response/unauthorized {})))
+
+      (api/POST "/start-kk-application-payment-status-updater-job-for-all" {session :session}
+        :path-params []
+        :summary "Triggers a job for updating internal payment status for all open higher education application payments"
+        (if (get-in session [:identity :superuser])
+          (do (kk-application-payment-status-updater-job/start-update-kk-payment-status-for-all-job
+                job-runner)
+              (response/ok {}))
+          (response/unauthorized {})))
+
+      (api/POST "/start-kk-application-payment-status-updater-job/:person-oid/:term/:year" {session :session}
+        :path-params [person-oid :- s/Str
+                      term :- s/Str
+                      year :- s/Str]
```

Comment:
Voi harkita kannattaisiko näitä jotenkin validoida

Reply:
Sinällään ei ole teknisesti kauheasti haittaa, vaikka jonoon menisi ajoja ei-valideilla parametreilla, mutta voisi kyllä olla käyttäjäystävällisyyden kantilta hyvä palauttaa heti virhe jos parametrit ovat väärät...